### DISCUSSION TITLE
How to build a minified viewer

### DISCUSSION THREAD

**Discussion by m-gallesio** [mozilla/pdf.js#17496] [Q&A]
I found the `gulp minified` command to produce a minified build of the base library, but this does not seem to output the *viewer*.
Moreover, just running `terser` (version 5.26.0) on `viewer.mjs` fails:
```
Parse error at .\viewer.mjs:15098,37
/******/ __webpack_exports__ = await __webpack_exports__;
                                     ^
ERROR: Unexpected token: name (__webpack_exports__)
```

Any help?

**Comment by bwaindwain** [comment]
It was removed. See https://github.com/mozilla/pdf.js/commit/8158628a9085c6c989e82c9b2631c8ecc819e0a7
